### PR TITLE
Migrated web notify

### PR DESCRIPTION
**Pull request by JayVora-SerpentCS**

### CODE REVIEW

File: web_notify/static/src/js/web_client.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ odoo.define('web_notify.WebClient', function (require) {
 
 var WebClient = require('web.WebClient');
 var base_bus = require('bus.bus');
-var _ = require('_');
+//var _ = require('_');
```
Comment (sbidoul) [inline web_notify/static/src/js/web_client.js]: Can you remove it instead of commenting out?

File: web_notify/__manifest__.py [hunk 2]
```diff
@@ -6,9 +6,11 @@
     'name': 'Web Notify',
     'summary': """
         Send notification messages to user""",
-    'version': '9.0.1.0.0',
+    'version': '10.0.1.0.0',
+    'description': 'Web Notify',
     'license': 'AGPL-3',
-    'author': 'ACSONE SA/NV,Odoo Community Association (OCA)',
+    'author': """Serpent Consulting Services Pvt. Ltd.,
+                ACSONE SA/NV, Odoo Community Association (OCA)""",
```
Comment (sbidoul) [inline web_notify/__manifest__.py]: Just a remark: I know there is no clear rule for that but I tend to think it's not fair to add your company as author when the work is a trivial migration like this one. It's perfectly fine to add you in the contributors list, though. What do you think?

File: web_notify/__manifest__.py [hunk 3]
```diff
@@ -6,9 +6,11 @@
     'name': 'Web Notify',
     'summary': """
         Send notification messages to user""",
-    'version': '9.0.1.0.0',
+    'version': '10.0.1.0.0',
+    'description': 'Web Notify',
     'license': 'AGPL-3',
-    'author': 'ACSONE SA/NV,Odoo Community Association (OCA)',
+    'author': """Serpent Consulting Services Pvt. Ltd.,
```
Comment (lmignon) [inline web_notify/__manifest__.py]: @JayVora-SerpentCS  Thank you for the migration but you are not the the author of this addons. IMO a change in the author can be done when you contribute to the module with a new functionality or a refactoring of the code not for a simple migration.

File: web_notify/static/src/js/web_client.js (hunk 1 above)
Comment (lmignon) [inline web_notify/static/src/js/web_client.js]: this commented line must be removed

**Review by lmignon — CHANGES_REQUESTED** [review]

**Comment by adrienpeiffer** [comment]
Hi @JayVora-SerpentCS,

Thanks for the work :smile: 
I just created a PR on your branch to add the setup structure for this module here https://github.com/JayVora-SerpentCS/web/pull/1

File: web_notify/__manifest__.py (hunk 2 above)
Comment (JayVora-SerpentCS) [reply web_notify/__manifest__.py]: We will do the needful. Good to clarify, thanks.

File: web_notify/__manifest__.py (hunk 3 above)
Comment (JayVora-SerpentCS) [reply web_notify/__manifest__.py]: Noted, we will update!

File: web_notify/__manifest__.py (hunk 2 above)
Comment (JayVora-SerpentCS) [reply web_notify/__manifest__.py]: We will do the needful. Good to clarify, thanks.

**Comment by JayVora-SerpentCS** [comment]
Please review again.
thanks.

**Review by rousseldenis — APPROVED** [review]
:+1: Code review

**Review by lmignon — APPROVED** [review]

File: web_notify/__manifest__.py [hunk 4]
```diff
@@ -6,9 +6,10 @@
     'name': 'Web Notify',
     'summary': """
         Send notification messages to user""",
-    'version': '9.0.1.0.0',
+    'version': '10.0.1.0.0',
+    'description': 'Web Notify',
     'license': 'AGPL-3',
-    'author': 'ACSONE SA/NV,Odoo Community Association (OCA)',
+    'author': """ACSONE SA/NV,Odoo Community Association (OCA)""",
```
Comment (lasley) [inline web_notify/__manifest__.py]: Please revert this change. The author tag should just be standard quote, not multiline

**Review by lasley — CHANGES_REQUESTED** [review]

**Review by lasley — APPROVED** [review]
Thanks @JayVora-SerpentCS

**Comment by rousseldenis** [comment]
@lmignon @lasley @adrienpeiffer @sbidoul Seems to have problems with web_enterprise addon (app switcher). Can someone confirm that ?

Sorry didn't noticed @adrienpeiffer comment

**Comment by adrienpeiffer** [comment]
@rousseldenis it doens't work without https://github.com/JayVora-SerpentCS/web/pull/2.
@JayVora-SerpentCS Can you consider this one ?

**Comment by JayVora-SerpentCS** [comment]
@rousseldenis I haven't checked, sorry.

**Review by adrienpeiffer — APPROVED** [review]
:+1:

**Comment by adrienpeiffer** [comment]
I think this one is ready to merge

**Review by sbidoul — APPROVED** [review]
